### DISCUSSION TITLE
[2기-B] 이상화

### DISCUSSION THREAD

**Discussion by fantasworm** [super-corini/programmers_kdt_II#357] [2기 Monthly Project I]
**Site: [http://ec2-13-125-39-148.ap-northeast-2.compute.amazonaws.com/](http://ec2-13-125-39-148.ap-northeast-2.compute.amazonaws.com/)**
**GitHub: [django-deploy-test](https://github.com/fantasworm/django-deploy-test.git)**

### 해결 중 어려웠던 점 및 극복 방법, 새로 공부한 것들

- 무료 버전에서 자주 용랑 폭발로 뻗어버리는 게 ㅠㅠ
- 난생 처음 리눅스 쓰는 거고 서버 관련 일을 하는 것이다 보니 정신없이 따라했습니다!
- 위와 같은 문제에도 불구하고 AWS를 쓰면서 커맨드 입력에 편리한 점들을 느끼면서 현 윈도우만 깔려 있는 컴퓨터에 리눅스 개발 환경을 만들어야 할 필요성을 배웠습니다.

**Comment by super-corini** [comment]
우분투나 맥 많이 쓰시더라구요. 추후 ML을 다루시게 된다면 아마 우분투를 많이 쓰시게 되실텐데, 이를 대비해서 CLI를 많이 경험해보시면 좋을 것 같습니다 😸
